cosmos/columbus/tokens/terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw.json

```diff
@@ -0,0 +1,9 @@
+{
+    "contractAddress": "terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw",
+    "imageUrl": "https://raw.githubusercontent.com/garudawork/keplr-contract-registry/main/images/columbus/GARUDAX.png",
```

Comment:
@garudawork Could you change repository name of imageUrl?

`https://raw.githubusercontent.com/garudawork`-> `https://raw.githubusercontent.com/chainapsis`